CodeEdit/Features/Editor/TabBar/Views/EditorTabBarTrailingAccessories.swift

```diff
@@ -69,11 +69,12 @@ struct EditorTabBarTrailingAccessories: View {
     func split(edge: Edge) {
         let newEditor: Editor
         if let tab = editor.selectedTab {
-            newEditor = .init(files: [tab])
+            newEditor = .init(files: [tab], temporaryTab: tab)
```

Comment:
I also don't really like opening the split tab as temporary. This isn't the behavior other editors have, and will lead to the same problem I mentioned in the `EditorView` comment.

Reply:
Again, this is patterned after Xcode. We can certainly provide a setting that changes this behavior though.

Reply:
That's true, we can talk more in discord though.